src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "11.3.0-prerelease",
+  "version": "11.4.0-prerelease",
```

Comment:
Since it removes all the pagination functions, it should be a major version upgrade, i.e. ```12.0.0-prerelease```.